service/src/main/scala/com/yahoo/maha/service/calcite/MahaCalciteSqlParser.scala

```diff
@@ -188,6 +191,7 @@ case class DefaultMahaCalciteSqlParser(mahaServiceConfig: MahaServiceConfig) ext
                   sqlIdentifier.names.asScala.foreach(
                     c => {
                       val publicCol: PublicColumn = getColumnFromPublicFact(publicFact, c, columnAliasToColumnMap)
+                      mahaSqlRequestContext.queryAliasToColumnNameMap += (publicCol.alias -> publicCol.alias)
```

Comment:
Add native method to mahaSqlRequestContext and add entry to map. It will be better to track

Reply:
Done, do review